src/components/NucleotideEntropy/NucleotideEntropyMultiChart.tsx

```diff
@@ -0,0 +1,196 @@
+import React, { ChangeEvent, Dispatch, SetStateAction, useState } from 'react';
+import Loader from '../Loader';
+import { LapisSelector } from '../../data/LapisSelector';
+import { PipeDividedOptionsButtons } from '../../helpers/ui';
+import { Legend, Line, LineChart, ResponsiveContainer, Tooltip, XAxis, YAxis } from 'recharts';
+import Checkbox from '@mui/material/Checkbox';
+import FormGroup from '@mui/material/FormGroup';
+import FormControlLabel from '@mui/material/FormControlLabel';
+import { SequenceType } from '../../data/SequenceType';
+import { Form } from 'react-bootstrap';
+import jsonRefData from '../../data/refData.json';
+import { GeneOption } from './calculateEntropy';
+import { useExploreUrl } from '../../helpers/explore-url';
+import { pprettyColors } from '../../helpers/colors';
+import { formatDate } from '../../widgets/VariantTimeDistributionLineChartInner';
+import { genes } from './genes';
+import { useMultiChartData } from './useMultiChartData';
+
+type Props = {
+  selectors: LapisSelector[];
+};
+
+function nonNullValuesToString(obj: object) {
+  return Object.entries(obj)
+    .filter(([_, value]) => value !== undefined)
+    .map(([_, value]) => value)
+    .toString();
+}
+
+export const NucleotideEntropyMultiChart = ({ selectors }: Props) => {
+  const [sequenceType, setSequenceType] = useState<SequenceType>('nuc');
+  const [selectedGene, setSelectedGene] = useState<string>('All');
+  const [includeDeletions, setIncludeDeletions] = useState<boolean>(false);
+
+  const handleDeletions = (event: React.ChangeEvent<HTMLInputElement>) => {
+    setIncludeDeletions(event.target.checked);
+  };
+
+  const exploreUrl = useExploreUrl()!;
+  const variants = (exploreUrl.variants ?? []).map(nonNullValuesToString);
```

Comment:
@chaoran-chen Does this particular line make sense? I'm not so sure about it. It seems to work, but it might be quite fragile.